### ISSUE TITLE
uvcdat-usage.llnl.gov broken in 1.1.2

### ISSUE THREAD

**Issue by durack1** [PCMDI/pcmdi_metrics#502]
@zshaheen it seems the logging is broken in `1.1.2`:
```
(PMP112) durack1@oceanonly:[ocn]:[4929]> python
Python 2.7.13 | packaged by conda-forge | (default, May  2 2017, 12:48:11)
[GCC 4.8.2 20140120 (Red Hat 4.8.2-15)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import pcmdi_metrics.io.base
>>> DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 200 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 200 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 200 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 200 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 200 None

KeyboardInterrupt
```

**Comment by zshaheen** [comment]
I think @doutriaux1 fixed this somewhere.

**Comment by doutriaux1** [comment]
@zshaheen I have a **partial** fix in https://github.com/PCMDI/pcmdi_metrics/pull/489 but you need to fix it better so that the log level is not set at the log module level but at the actual log file level.

**Comment by durack1** [comment]
@zshaheen @doutriaux1 I've just hit the same issue today:
```
durack1@oceanonly:[~]:[6147]> source activate PMP112
(PMP112) durack1@oceanonly:[~]:[6113]> python
Python 2.7.13 | packaged by conda-forge | (default, May  2 2017, 12:48:11)
[GCC 4.8.2 20140120 (Red Hat 4.8.2-15)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from pcmdi_metrics.io.base import generateProvenance
>>> DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 200 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 200 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 200 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 200 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 500 None
DEBUG:urllib3.connectionpool:https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 200 None

KeyboardInterrupt
```
Do we need to push out a 1.1.3 to fix this issue? If yes, it'd be great to do that ASAP, as we're trying to get some more data generated for GFDL

**Comment by doutriaux1** [comment]
nighlty cdat_info should fix this

**Comment by durack1** [comment]
@doutriaux1 @zshaheen how do I hook up a PMP112 with cdat_info nightly?

We really need to make sure that out of the box this stuff just works, so I would push for a 1.1.3 to sort things out so that the issue is resolved. At the moment I assume any user that installs 1.1.2 will hit exactly the same issue I am reproducing time and time again

**Comment by doutriaux1** [comment]
I have also a fix independent of cdat_info in the branch I'm reviewing for @gleckler1 which I'm hoping to push in today.

**Comment by doutriaux1** [comment]
thinking about it I can edit the above tweak and use the one in cdat_info. @zshaheen no need to look into it any longer.

**Comment by gleckler1** [comment]
@durack1 @doutriaux1 @zshaheen  This looks familiar.   After an install I got a prompt for something like do I want to report debug stuff.   When I said yes it bombed like PDs case.  But when I said no everything ran fine.

**Comment by durack1** [comment]
@doutriaux1 perfect, once this is in I'll pull down and make sure I'm fixed and we can tag the 1.1.3. It's important that we clean this right up.

@gleckler1 please comment if you don't agree

**Comment by lee1043** [comment]
I am getting same issue, @doutriaux1 @durack1 @gleckler1 how this was concluded?

**Comment by doutriaux1** [comment]
It's awaiting me tomorrow. Sorry @lee1043 I'm working on your taylordiagrams today.

**Comment by doutriaux1** [comment]
guys in the mean time in yuor scripts add:
```python
import logging
LOG_LEVEL = logging.INFO
logging.basicConfig(level=LOG_LEVEL)
```

**Comment by lee1043** [comment]
@doutriaux1 that helps, thanks!

**Comment by durack1** [comment]
@doutriaux1 we really need to fix this soon, v1.1.3 should bump us across to UV-CDAT 2.12

**Comment by durack1** [comment]
This problem is solved by updating `cdat_info` to 2.12+
